maven-plugins/pom.xml

```diff
@@ -145,9 +148,21 @@ under the License.
 
   <reporting>
     <plugins>
+       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <reportSets>
+          <reportSet>
+            <id>default</id>
+            <reports>
+              <report>javadoc</report>
+            </reports>
+          </reportSet>
+        </reportSets>
+      </plugin>
```

Comment:
we already have such report in maven-parent, 
it is in reporting profile ... so maybe also maven-plugin-report-plugin should be in reporting profile

Reply:
why the separate profile? What is the site lifecycle supposed to generate without that profile?

Reply:
Separate profile only to be consistent how we do it now.

It is, what I mention from time to time - I think that we have to many reports, and we introduce one simple site build and one with full reports with profile - but consensus is difficult to achieve.
Sometime during release special profile can be forgotten....

Reply:
Ok, so what about moving all reports (incl. `maven-plugin-report-plugin` and `maven-project-info-reports-plugin`) to the profile `reporting` and always activate that profile during (in addition to `apache-release`) in `release:perform` (https://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html#releaseProfiles)? Then only the manually maintained pages are built during a regular `site` execution.

Reply:
Done in https://github.com/apache/maven-parent/pull/90/commits/82990e3a4c6c0f9896a0a32dd66d58ef0ee7b670